cg/meta/orders/status.py

```diff
@@ -372,8 +372,8 @@ def store_fastq_samples(
                     raise OrderError(f"Invalid application: {sample['application']}")
                 new_sample.application_version = application_version
                 new_samples.append(new_sample)
-                data_analysis: Pipeline = self.if_type_wgs_do_mip_dna_analysis_for_maf(
-                    application_version
+                data_analysis: Pipeline = self.get_fastq_pipeline(
```

Comment:
What about a mixture of tumour and normals? Will this code handle that or will the pipeline be set for the last sample processed?

Reply:
We create on case per fastq sample so it should be fine even in the case of a mixture